File: src/API/WesternStatesWater.WestDaat.Contracts.Client/PointEtDataCollection.cs
namespace WesternStatesWater.WestDaat.Contracts.Client;

public class PointEtDataCollection
{
    public Guid WaterConservationApplicationEstimateControlLocationId { get; set; }

    public string PointWkt { get; set; }

    public double AverageYearlyTotalEtInInches { get; set; }

    public GeometryEtDatapoint[] Datapoints { get; set; }
}

File: src/DashboardUI/src/data-contracts/PointEtDataCollection.ts
import { GeometryEtDatapoint } from './GeometryEtDatapoint';

export interface PointEtDataCollection {
  waterConservationApplicationEstimateControlLocationId: string;
  pointWkt: string;
  averageYearlyTotalEtInInches: number;
  datapoints: GeometryEtDatapoint[];
}
